src/main/webapp/app/shared/pipes/html-for-markdown.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ArtemisMarkdown } from 'app/components/util/markdown.service';
+
+@Pipe({
+    name: 'htmlForMarkdown',
+})
+export class HtmlForMarkdownPipe implements PipeTransform {
```

Comment:
Very neat! In the long run, we should use this pipe everywhere in Artemis instead of the `formattedXY` properties! :+1: